Automattic/wp-calypso
# Stats Widget: Rework the module cards

Related to #75105, #75104 

## Proposed Changes

* Add module cards of `Akismet` and `Brute force protection` to the Stats Widget.
* Add the Jetpack logo as the footer.

<img width="601" alt="截圖 2023-04-13 上午12 12 24" src="https://user-images.githubusercontent.com/6869813/231518399-a0c419c3-db06-44d8-b3d7-e3d8dbf1ab5b.png">

## Testing Instructions



* Open your local Jetpack site.
* Check out the branch of the Jetpack to the latest `trunk`.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin/index.php?odyssey_widget=1`.
* Ensure the module cards and the Jetpack logo at the bottom of the Stats Widget are in line with the design.

## Pre-merge Checklist
